ddtrace/tracer/metrics_test.go

```diff
@@ -87,6 +89,125 @@ func TestEnqueuedTracesHealthMetric(t *testing.T) {
 	assert.Equal(uint32(0), w.tracesQueued)
 }
 
+func TestSpansStartedTags(t *testing.T) {
+	var tg statsdtest.TestStatsdClient
+
+	defer func(old time.Duration) { statsInterval = old }(statsInterval)
+	statsInterval = time.Millisecond
+
+	t.Run("default", func(t *testing.T) {
+		assert := assert.New(t)
+		tracer, _, _, stop := startTestTracer(t, withStatsdClient(&tg))
+		defer stop()
+
+		tracer.StartSpan("operation").Finish()
+		tg.Wait(assert, 1, 100*time.Millisecond)
+
+		counts := tg.Counts()
+		assert.Equal(counts["datadog.tracer.spans_started"], int64(1))
+		for _, c := range statsdtest.FilterCallsByName(tg.CountCalls(), "datadog.tracer.spans_started") {
+			if slices.Equal(c.Tags(), []string{"integration:manual"}) {
+				return
+			}
+		}
+		assert.Fail("expected integration:manual tag in spans_started")
+	})
+
+	t.Run("other_source", func(t *testing.T) {
```

Comment:
same comment as on mockspan_test.go; let's use the term "integration" or "component" rather than source?